src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +33,19 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
+      
```

Comment:
Nagyon jó észrevétel, hogy a kivételt nem csak létrehozod, hanem el is dobod, good job.